QModManager/PirateCheck.cs

```diff
@@ -110,7 +123,7 @@ private void ShowText()
             {
                 DestroyImmediate(gameObject.GetComponent<RawImage>());
                 Text text = gameObject.AddComponent<Text>();
-                text.text = $"Oopsie poopsie, an error has occured!\nQModManager couldn't be initialized.\nTo fix this issue, turn on your internet connection,\nthen restart {(Patcher.game == Patcher.Game.Subnautica ? "Subnautica" : "Below Zero")}.\n(Is the game pirated?)";
+                text.text = $"An error has occured!\nQModManager couldn't be initialized.\nPlease turn on your internet connection,\nthen restart {(Patcher.game == Patcher.Game.Subnautica ? "Subnautica" : "Below Zero")}.";
```

Comment:
Don't use `\n` for line breaks in text.  
Use `Environment.Newline` instead which is safer and will always use the correct line break characters for the current OS.

Reply:
`\n` is used in soo many places now, and I think that there isn't an actual need to change it. Windows uses `\r\n` for line breaks _for backward compatibility_, and most other operating systems use `\n`. Despite that, the `\n` character behaves like `\r\n` on Windows operating systems, meaning that there is no actual need to change it. (IN MY OPINION)

Reply:
Wasn't there some sort of incompatibility because of using `\n` on other operation systems? I remember there was. But in any case, Environment.NewLine is definitely better.

Reply:
`\r\n` is only used for Unix based operating systems, such as Windows.
`\n` is used by all other operating systems, including Mac.

Quote from [here](https://stackoverflow.com/questions/1015766/difference-between-n-and-environment-newline):
> Note that when writing to the console, `Environment.NewLine` is not strictly necessary. The console stream will translate `\n` to the appropriate new-line sequence, if necessary.

`Environment.NewLine` just seems messy to me, creating very long lines or strings that span across multiple lines. So far there haven't been any errors with `\n` on _any_ operating systems, but if there are, I will definitely change it.